.github/CONTRIBUTING.md

```diff
@@ -11,6 +11,7 @@ CONTRIBUTING
 - [Accessibility](#accessibility)
   - [Role and aria props](#role-and-aria-props)
   - [Focus](#focus)
+  - [Keyboard handling](#keyboard-handling)
```

Comment:
not sure how this is here

Reply:
it's because this change always shows up in the git changelog whenever we switch to a new branch... a lot of us are experiencing the same... @alinais any ideas why it's happening?

Reply:
There is a script `yarn build:docs` which eventually runs `gulp build:docs:toc`.  That gulp task is responsible for generating the table of contents for markdown files.

If you change a markdown file's headings, the table of contents will be out of date.  Since we don't run doc builds prepush (it is super slow), there is a chance that you can edit a markdown's headings and merge a PR without having first run a doc build in your PR.  A subsequent PR will likely update since users run the docs after pulling the latest master changes for the base of their PR.

### The fix

We could introduce some logic to check if markdown files were changed and if so, run `gulp build:docs:toc` to regenerate the table of contents.  Low priority and not that big of an issue at present.  Let's save this discussion and work for another PR.